sdk/src/main/java/com/bugsnag/android/ExceptionHandler.java

```diff
@@ -91,7 +92,7 @@ public void uncaughtException(@NonNull Thread thread, @NonNull Throwable throwab
             originalHandler.uncaughtException(thread, throwable);
         } else {
             System.err.printf("Exception in thread \"%s\" ", thread.getName());
-            throwable.printStackTrace(System.err);
+            Logger.warn("Exception", throwable);
```

Comment:
Android redirects stderr to its own logger, but there's a limit on the message size which means larger methods won't be logged. `Logger` is also disabled by default in production, where it might not make sense to log this information.